model/layout.md

```diff
@@ -71,6 +72,24 @@ When layout direction is `"column"`:
 
 **Kind**: instance property of [<code>Layout</code>](#Layout)
 
+<a name="Layout+distribution"></a>
+### layout.distribution : <code>String</code>
+Determines how the children are distributed inside the layer.
+
+When layout direction is `"row"`:
+- `"center"`: 
+- `"max"`: 
+- `"min"`: 
+- `"space-between"`: 
+
+When layout direction is `"column"`:
```

Comment:
```suggestion
When layout direction is `column`:
```

Reply:
This is how it's used in other places in the repo, so I'll stick with it.